src/index.js

```diff
@@ -55,44 +56,47 @@ const applyCommit = (state, commitId, reducer) => {
       meta: Object.assign({}, actionToCommit[1].meta,
```

Comment:
This will fail, but there's no test currently that tests for it (Immutable `findEntry` returns an array of `[index, entry]`, native `find` just returns the entry).

Need to cherry-pick the new tests from #33 (or wait for that to merge first).